batch/batch/worker/worker.py

```diff
@@ -113,20 +113,18 @@ async def wrapper(f, *args, **kwargs):
             except DockerError as e:
                 # 408 request timeout, 503 service unavailable
                 if e.status == 408 or e.status == 503:
-                    log.exception(f'in docker call to {f.__name__} for {name}, retrying', stack_info=True)
+                    log.warning(f'in docker call to {f.__name__} for {name}, retrying', stack_info=True, exc_info=True)
                 # DockerError(500, 'Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
                 # DockerError(500, 'error creating overlay mount to /var/lib/docker/overlay2/545a1337742e0292d9ed197b06fe900146c85ab06e468843cd0461c3f34df50d/merged: device or resource busy'
                 # DockerError(500, 'Get https://gcr.io/v2/: dial tcp: lookup gcr.io: Temporary failure in name resolution')
                 elif e.status == 500 and ("request canceled while waiting for connection" in e.message
                                           or re.match("error creating overlay mount.*device or resource busy", e.message)
                                           or "Temporary failure in name resolution" in e.message):
-                    log.exception(f'in docker call to {f.__name__} for {name}, retrying', stack_info=True)
+                    log.warning(f'in docker call to {f.__name__} for {name}, retrying', stack_info=True, exc_info=True)
                 else:
                     raise
-            except asyncio.CancelledError:  # pylint: disable=try-except-raise
-                raise
```

Comment:
this isn't a subtype of any of the other caught errors, so there's no need to catch and raise